Sources/Fuzzilli/Mutators/OperationMutator.swift

```diff
@@ -135,6 +135,18 @@ public class OperationMutator: BaseInstructionMutator {
                 }
             }
             newOp = DestructArrayAndReassign(indices: newIndices.sorted(), hasRestElement: !op.hasRestElement)
+        case let op as DestructObject:
+            var newProperties = Set(op.properties)
+            if newProperties.count > 0 {
+                newProperties.remove(newProperties.randomElement()!)
+            }
```

Comment:
Shouldn't this also add a new property to the set (here and below)?

Also now that I see it, in the two cases above (`DestructArray`, `DestructArrayAndReassign`), let's try to get rid of the `while` loop: in case of an imported program (compiled from JS, for example), we could end up with a `DestructArray` that has indices 0 through 20 or so, and then this would turn into an infinite loop :(

How about we make the logic for these four cases something like
- remove a random element from the set
- attempt X times (let's say 10) to insert a random new element, then just give up